spec/bas/read/github/repo_issues_spec.rb

```diff
@@ -38,24 +38,24 @@
       allow(Octokit::Client).to receive(:new).and_return(octokit)
     end
 
-    it "fetch issues from the Github repo when there are no 'issues'" do
+    it "read issues from the Github repo when there are no 'issues'" do
       allow(octokit).to receive(:public_send).and_return(empty_response)
 
-      fetched_data = @fetcher.fetch
+      readed_data = @read.execute
 
-      expect(fetched_data).to be_an_instance_of(Fetcher::Github::Types::Response)
-      expect(fetched_data.results).to be_an_instance_of(Array)
-      expect(fetched_data.results.length).to eq(0)
+      expect(readed_data).to be_an_instance_of(Read::Github::Types::Response)
+      expect(readed_data.results).to be_an_instance_of(Array)
+      expect(readed_data.results.length).to eq(0)
     end
 
-    it "fetch issues from the Github repo when there are 'issues'" do
+    it "read issues from the Github repo when there are 'issues'" do
       allow(octokit).to receive(:public_send).and_return(response)
 
-      fetched_data = @fetcher.fetch
+      readed_data = @read.execute
```

Comment:
🤔 I think the read verb works for both past participle/tense wo I think the correct name should be:
```suggestion
      read_data = @read.execute
```